src/views/Accessibility/AccessibiltyRequest/Documents/New/index.tsx

```diff
@@ -125,49 +159,179 @@ const New = () => {
+                                      value={commonType}
+                                    />
+                                  );
+                                }
+                              )}
+                              <Field
+                                as={RadioField}
+                                checked={
+                                  values.documentType.commonType === 'OTHER'
+                                }
+                                id="FileUpload-CommonTypeOTHER"
+                                name="documentType.commonType"
+                                label={getDocumentTypeLabel(
+                                  AccessibilityRequestDocumentCommonType.OTHER
+                                )}
+                                onChange={() => {
+                                  setFieldValue(
+                                    'documentType.commonType',
+                                    'OTHER'
+                                  );
+                                }}
+                                value="OTHER"
+                              />
+                              {values.documentType.commonType === 'OTHER' && (
+                                <div className="width-card-lg margin-left-4 margin-bottom-1">
+                                  <FieldGroup
+                                    scrollElement="documentType.otherType"
+                                    error={
+                                      !!flatErrors['documentType.otherType']
+                                    }
+                                  >
+                                    <Label
+                                      htmlFor="FileUpload-OtherType"
+                                      className="margin-bottom-1"
+                                      style={{ marginTop: '0.5em' }}
+                                    >
+                                      Document name
+                                    </Label>
+                                    <FieldErrorMsg>
+                                      {flatErrors['documentType.otherType']}
+                                    </FieldErrorMsg>
+                                    <Field
+                                      as={TextField}
+                                      error={
+                                        !!flatErrors['documentType.otherType']
+                                      }
+                                      className="margin-top-0"
+                                      id="DocumentType-OtherType"
+                                      name="documentType.otherType"
+                                    />
+                                  </FieldGroup>
+                                </div>
+                              )}
+                            </fieldset>
+                          </FieldGroup>
+                          <div className="padding-top-2">
+                            <p>
```

Comment:
The width restriction on `Form` is restricting the width on this piece of text.

It might be easy fix, but I remember having issues with submission and validations when the submit button is outside of `Form`.

Let's make this a stretch goal and deal with this styling bug **after** this PR and functionality is complete.